docs_source/content/resource-authors-guide.md

```diff
@@ -86,6 +86,14 @@ these rules:
 1. Otherwise, if there are any diffs which say that they contain a difference,
    the Status will always show as having changes.
 
+### Dealing with Errors
+
+The default `Status` implementation has a `SetError(error)` method. When called,
+it sets an appropriate error level and an internal error state that will be
+presented to the user. You can also use the common Go patterm of returning `nil,
+err` in your `Check` and `Apply` statements. Converge will call `SetError(err)`
+automatically in this case.
+
```

Comment:
fyi, until #345 is merged, your error can be lost (regardless of whether you `SetError` or just return it) if the final plan phase `HasChanges`